ui/components/multichain/pages/review-permissions-page/review-permissions-page.tsx

```diff
@@ -145,28 +178,84 @@ export const ReviewPermissions = () => {
     setShowNetworkToast(true);
   };
 
-  const accounts = useSelector(getUpdatedAndSortedAccounts);
-  const evmAccounts: MergedInternalAccount[] = useMemo(() => {
-    return accounts.filter((account: InternalAccount) =>
-      isEvmAccountType(account.type),
-    );
-  }, [accounts]);
+  const allAccounts = useSelector(
+    getUpdatedAndSortedAccountsWithCaipAccountId,
+  ) as MergedInternalAccountWithCaipAccountId[];
 
-  const connectedAccountAddresses = useSelector((state) =>
-    getPermittedAccountsForSelectedTab(state, activeTabOrigin),
-  ) as string[];
+  const _connectedAccountAddresses = useSelector((state) =>
```

Comment:
Maybe this can be called `nonRemappedConnectedAccountAddresses`?

Reply:
Done here: https://github.com/MetaMask/metamask-extension/pull/29887/commits/8c2973e3a2e21ea9b878b39c14e94c3fb80cdb6a